lib/talkable/generators/install_generator.rb

```diff
@@ -0,0 +1,56 @@
+module Talkable
+  class InstallGenerator < Rails::Generators::Base
+    source_root File.expand_path("../templates", __FILE__)
+    class_option :haml, type: :boolean, default: false
+    class_option :slim, type: :boolean, default: false
+
+    def ask_config_values
+      @site_slug  = ask("Your Talkable site slug:")
+      @api_key    = ask("Your Talkable API Key:")
+      if ['', 'y', 'Y'].include?(ask 'Do you have a custom domain? [Y/n]')
+        @server   = ask("Your custom domain [#{Talkable::Configuration::DEFAULT_SERVER}]:")
+      end
+    end
+
+    def add_initializer
+      template "config/initializers/talkable.rb", "config/initializers/talkable.rb"
+    end
+
+    def inject_talkable_offer
```

Comment:
I don't like either, sometime it looks prettier but most of all it only confuses.